evaluation/humanevalfix/README.md

```diff
@@ -0,0 +1,74 @@
+# HumanEvalFix Evaluation with OpenDevin
+
+Implements evaluation of agents on HumanEvalFix from the HumanEvalPack benchmark introduced in [OctoPack: Instruction Tuning Code Large Language Models](https://arxiv.org/abs/2308.07124). Please see https://github.com/bigcode-project/bigcode-evaluation-harness/blob/main/bigcode_eval/tasks/humanevalpack.py for the reference implementation used in the paper.
+
+## Setup Environment
+
+Please follow [this document](https://github.com/OpenDevin/OpenDevin/blob/main/Development.md) to setup local develop environment for OpenDevin.
+
+In addition, evaluation requires the `evaluate` package installable via:
+```bash
+pip install evaluate
+```
+
+## Configure OpenDevin and your LLM
+
+Create a `config.toml` file if it does not exist at the root of the workspace.
+
+Add the following configurations:
+
+```toml
+[core]
+max_iterations = 100
+cache_dir = "/tmp/cache"
+ssh_hostname = "localhost"
```

Comment:
You probably also want to include `enable_auto_lint = true`. Evaluation of CodeActAgent on `SWE-bench-lite` shows that this option could give the LLM a hint of indentation errors, and thus boosts the final score.